hiku/telemetry/prometheus.py

```diff
@@ -153,15 +146,23 @@ def visit_link(self, obj):
 class _SubqueryMixin:
 
     def subquery_wrapper(self, observe, subquery):
-        def wrapper(field_names, *args):
+        def wrap_proc(start_time, field_name, proc):
+            def _proc_wrapper(*args):
+                result = proc(*args)
```

Comment:
passing start_time from above allows to track subgraph execution + proc execution time.

Reply:
maybe it makes sense to place `start_time = time.perf_counter()` inside `_proc_wrapper` but then we are loosing (probably) subgraph(low-level graph) data exection time